packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -6,43 +6,126 @@ import {
   MEETINGS
 } from '../constants';
 
+/**
+ * IInMeetingActions
+ * Type for In-Meeting Actions
+ */
+interface IInMeetingActions {
+  canInviteNewParticipants: boolean | null;
+
+  canAdmitParticipant: boolean | null;
+
+  canLock: boolean | null;
+
+  canUnlock: boolean | null;
+
+  canAssignHost: boolean | null;
+
+  canStartRecording: boolean | null;
+
+  canPauseRecording: boolean | null;
+
+  canResumeRecording: boolean | null;
+
+  canStopRecording: boolean | null;
+
+  canRaiseHand: boolean | null;
+
+  canLowerAllHands: boolean | null;
+
+  canLowerSomeoneElsesHand: boolean | null;
+
+  canEnableClosedCaption: boolean | null;
+
+  canStartTranscribing: boolean | null;
+
+  canStopTranscribing: boolean | null;
+
+  isClosedCaptionActive: boolean | null;
+
+  isWebexAssistantActive: boolean | null;
+
+  canViewCaptionPanel: boolean | null;
+
+  isRealTimeTranslationEnabled: boolean | null;
+
+  canSelectSpokenLanguages: boolean | null;
+}
+
 /**
  * @class InMeetingActions
  */
 export default class InMeetingActions {
```

Comment:
you can use `IInMeetingActions` interface here:
```suggestion
export default class InMeetingActions implements IInMeetingActions{
```

Reply:
Nice one